<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1032</width>
    <height>711</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1032</width>
    <height>711</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gitea GUI</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1011</width>
     <height>481</height>
    </rect>
   </property>
   <property name="indentation">
    <number>30</number>
   </property>
   <property name="headerHidden">
    <bool>false</bool>
   </property>
   <attribute name="headerDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="headerHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件类型</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件大小</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>仓库创建者</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>默认分支</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>590</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="log_textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>841</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_name_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string> 输入要搜索的仓库名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_button">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>刷新列表</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
